pagefaultgames/pokerogue
# [Qol] Load i18n en locales during tests

## Why am I making these changes?

Right now it's kinda hard to read the obfuscated logs during tests as it's just i18n keys.

## What are the changes from a developer perspective?

- Added a MSW that handles loading the locales files during tests
- Fix all tests that were written in a way that they were affected by this. They shouldn't anymore in the future now.

### Screenshots/Videos

N/A

## How to test the changes?

Run the tests. YOu should see human readable english output yet fully functional tests.

## Checklist
- ~~[] Are the changes visual?~~
